js/store.js

```diff
@@ -11,6 +10,91 @@ const store = {
       }
     }
     browser.runtime.onMessage.addListener((m) => store.messageHandler(m));
+
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList;
```

Comment:
@biancadanforth best to move this whitelist related part into a separate function. its nice to keep in `init` only variable assignments and further function calls.